resource-detectors/lib/opentelemetry/resource/detectors.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
```

Comment:
New files should have 2020. There are a number of files with 2019. We can also considering removing the year (as a separate issue). The JS SIG recently did this: https://github.com/open-telemetry/opentelemetry-js/pull/1150.